install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,425 @@
 ////
 Module included in the following assemblies:
-
 install_config/configuring_rhv.adoc
 ////
-
 [id='configuring-rhv-objects_{context}']
-= Configuring Red Hat Virtualization objects
+= Configuring Red Hat Virtualization
+
+To integrate {product-title} with Red Hat Virtualization, perform the following actions as part of your xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+
+== Creating the bastion virtual machine
+
+The bastion virtual machine is used to install {product-title} on Red Hat Virtualization.
+
+.Procedure
+
+. Log in to the Manager machine using SSH.
+. Create a temporary bastion installation directory, for example, *_/bastion_installation_*, for the installation files.
+. Create an encrypted file, *_/bastion_installation/secure_vars.yaml_*:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-vault create secure_vars.yaml
+----
+
+. Add the following variables to *_secure_vars.yaml_*:
++
+[source,yml]
+----
+engine_password: <Manager_password>
+bastion_root_password: <bastion_root_password>
+rhsub_user: <Red_Hat_Subscription_Manager_username>
+rhsub_pass: <Red_Hat_Subscription_Manager_password>
+rhsub_pool: <Red_Hat_Subscription_Manager_pool_id>
+root_password: <OpenShift_node_root_password>
+engine_cafile: <Engine_CA_certificate>
+oreg_auth_user: <image_registry_authentication_username>
+oreg_auth_password: <image_registry_authentication_password>
+----
+
+. Save the file and record the vault password.
+
+. Create the installation playbook, *_/bastion_installation/create-bastion-machine-playbook.yaml_*, and update its parameters:
++
+[source,yml]
+----
+---
+- name: Create a bastion machine
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  roles:
+    - oVirt.image-template
+    - oVirt.vm-infra
+  no_log: true
```

Comment:
@apinnick @rgolangh I am sorry, I was not explicit enough when requesting this change. The `roles` section should have stayed where it was, it was in logical place. I wanted just the `no_log` section to move under `gather_facts`. The functionality will remain unchanged. This is basically just a step towards better clarity. The `no_log: true` with current indentation applies to the whole play, so there's no real reason why should it be after `roles` statement. It looks somewhat confusing and to have it after `roles` statement when it applies to the same scope as `connection` or `gather_facts`.

Reply:
Done. Please check.

Reply:
Looks good!